broker/test/test_broker/mocks/cloudController.js

```diff
@@ -176,12 +176,9 @@ function getServicePlans(broker_guid, plan_guid, plan_unique_id) {
     });
 }
 
-function getSpaces(broker_guid, space_guid) {
+function getSpaces(space_guid) {
```

Comment:
`broker_guid` is not  used in `getSpaces()` and `getOrganizations()` methods. Removing nock query method  validating request with query string ` q: service_broker_guid:${broker_guid}`.